datasets/panstarrs.yaml

```diff
@@ -0,0 +1,24 @@
+Name: Pan-STARRS PS1 Survey
+Description: |
+  The PS1 surveys used a 1.8 meter telescope and its 1.4 Gigapixel camera to image the sky in five broadband filters. The largest of these surveys provides coverage of the entire sky north of -30 degrees declination.
+Documentation: https://outerspace.stsci.edu/display/PANSTARRS/
+Contact: archive@stsci.edu
+ManagedBy: "[Space Telescope Science Institute](http://www.stsci.edu/)"
+Citation: Please see the documentation for full citation instructions.
+UpdateFrequency: Never
+Tags:
+  - astronomy
+License: STScI hereby grants the non-exclusive, royalty-free, non-transferable, worldwide right and license to use, reproduce, and publicly display in all media data from the PS1 surveys.
+Resources:
+  - Description: PS1 DR1 and DR2 files
+    ARN: arn:aws:s3:::stpubdata/ps1
+    Region: us-east-1
+    Type: S3 Bucket
+    RequesterPays: False
+DataAtWork:
+  Tutorials:
+  - Title: Light Curves with PanSTARRS
+    URL: https://spacetelescope.github.io/mast_notebooks/notebooks/PanSTARRS/PS1_DR2_TAP/PS1_DR2_TAP.html
```

Comment:
As you said, this notebook does not use the S3 data (or even the images) at all, so it is marginally relevant.  Using this link as a placeholder is OK with me though.